third_party/3/bleach/linkifier.pyi

```diff
@@ -0,0 +1,30 @@
+from typing import Any, List, MutableMapping, Protocol, Pattern, Iterable, Optional
+
+_Attrs = MutableMapping[Any, str]
+
+class _Callback(Protocol):
+    def __call__(self, attrs: _Attrs, new: bool = ...) -> _Attrs: ...
+
+DEFAULT_CALLBACKS: List[_Callback]
+
+TLDS: List[str]
+
+def build_url_re(tlds: Iterable[str] = ..., protocols: Iterable[str] = ...) -> Pattern[str]: ...
+
+URL_RE: Pattern[str]
+PHOTO_RE: Pattern[str]
+EMAIL_RE: Pattern[str]
+
+class Linker:
```

Comment:
Should inherit from object.

However, I just noticed that the stubs are in third_party/3. Why is that? The library looks like it's 2/3-compatible.